_query-dsl/specialized/more-like-this.md

```diff
@@ -0,0 +1,286 @@
+
+Saving [`term_vector`]({{site.url}}{{site.baseurl}}/field-types/mapping-parameters/term-vector/) information when indexing documents can greatly accelerate `more_like_this` queries because the engine can directly retrieve the important terms without reanalyzing the field text at query time.
+{: .note}
+
+## Example: No term vector optimization
+
+Create an index named `articles-basic` using the following mapping:
+
+```json
+PUT /articles-basic
+{
+  "mappings": {
+    "properties": {
+      "title": { "type": "text" },
+      "content": { "type": "text" }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Add sample documents:
+
+```json
+POST /articles-basic/_bulk
+{ "index": { "_id": 1 }}
+{ "title": "Exploring the Sahara Desert", "content": "Sand dunes and vast landscapes." }
+{ "index": { "_id": 2 }}
+{ "title": "Amazon Rainforest Tour", "content": "Dense jungle and exotic wildlife." }
+{ "index": { "_id": 3 }}
+{ "title": "Mountain Adventures", "content": "Snowy peaks and hiking trails." }
+```
+{% include copy-curl.html %}
+
+Query using the following request:
+
+```json
+GET /articles-basic/_search
+{
+  "query": {
+    "more_like_this": {
+      "fields": ["content"],
+      "like": "jungle wildlife",
+      "min_term_freq": 1,
+      "min_doc_freq": 1
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The `more_like_this` query searches for the terms `jungle` and `wildlife` in the `content` field, which matches only one document:
+
+```json
+{
+  ...
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 1.9616582,
+    "hits": [
+      {
+        "_index": "articles-basic",
+        "_id": "2",
+        "_score": 1.9616582,
+        "_source": {
+          "title": "Amazon Rainforest Tour",
+          "content": "Dense jungle and exotic wildlife."
+        }
+      }
+    ]
+  }
+}
+```
+
+## Example: Term vector optimization
+
+Create an index named `articles-optimized` using the following mapping:
+
+```json
+PUT /articles-optimized
+{
+  "mappings": {
+    "properties": {
+      "title": {
+        "type": "text",
+        "term_vector": "with_positions_offsets"
+      },
+      "content": {
+        "type": "text",
+        "term_vector": "with_positions_offsets"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Insert sample documents into the optimized index:
+
+```json
+POST /articles-optimized/_bulk
+{ "index": { "_id": "a1" } }
+{ "name": "Diana", "alias": "Wonder Woman", "quote": "Justice will come when it is deserved." }
+{ "index": { "_id": "a2" } }
+{ "name": "Clark", "alias": "Superman", "quote": "Even in the darkest times, hope cuts through." }
+{ "index": { "_id": "a3" } }
+{ "name": "Bruce", "alias": "Batman", "quote": "I am vengeance. I am the night. I am Batman!" }
+```
+{% include copy-curl.html %}
+
+Find documents similar to `dark` and `night` using the following request:
```

Comment:
@AntonEliatra The documents themselves can't literally be similar to `dark` and `night`. Do we mean "Find documents similar to those containing `dark` and `night`..."?

Reply:
@natebower I've updated this now to:

Find documents where the `quote` field contains terms similar to "dark" and "night"